### ISSUE TITLE
Latency is Extreamly High

### ISSUE THREAD

**Issue by Batcastle** [hensm/fx_cast#63]
When casting on Wi-Fi latency is extremely high (about 2 seconds). This of course will vary with internet speeds, but with the internet speeds I deal with, normal latency when casting from Android or a Chromium-based browser is usually below a quarter of a second. Issue found on YouTube, VRV, and Netflix.

**Comment by hensm** [comment]
Is this initial connection latency or does it persist throughout the lifetime of the session?

If so, when you make changes that result in new state being pushed to the sender application (e.g. volume change in Google Home changing the web page volume slider), is there a similar delay?

**Comment by Batcastle** [comment]
Yes, this persists throughout the entire session and state changes are also delayed. I know there was one point I went to pause something casting from Netflex and it became un-synced (the point it paused on the receiving device WAS NOT where I paused on the sending device).
